### DISCUSSION TITLE
BlueSCSI with Mac IIfx

### DISCUSSION THREAD

**Discussion by rsnorkel** [erichelgeson/BlueSCSI-v1#200] [Q&A] [answered]
Wondering whether BlueSCSI would work on my Mac IIfx? It seems that that this model had some special SCSI termination requirements. I used a Quantum Lightning 730S that was internally terminated so as long as the BlueSCSI is terminated I would think it would work. Definitely would like feedback on this. Thanks.

**Comment by bakkus** [comment, marked as answer]
I have the BlueSCSI working with my IIfx.
What I noticed is that I need the BlueSCSI as the last device on the bus with powered termination using the Berg connector.
Any other combo results in intermittent termination and therefore intermittent booting/disk mounting/etc.

**Reply by rsnorkel** [reply]
Thank you for the information. When you say that the BlueSCSI is the last device on the SCSI bus can I assume that you are using it as an external drive? If so have you tried to use it as an internal drive replacement?

**Reply by erichelgeson** [reply]
Chain can be internal or external - you can have multiple internal scsi devices and multiple external - the last device on each end should be terminated. Berg power connector is only on the 50pin internal device, I'd assume he's using it internally.

**Reply by rsnorkel** [reply]
Just to close this, I purchased a BlueSCSI and it works perfectly on my mac iifx. I am not using the terminator block that shipped with the mac iifx motherboard when I upgraded my original mac ii. I do have the "Berg" connector attached but I am not sure whether my system needs it. The BlueSCSI appears to be powered by the external CDROM drive that is attached. I may experiment with removing it and see what happens.
